CHANGELOG.md

```diff
@@ -7,6 +7,48 @@ Entries in this file should be limited to:
 - Obscure side-effects that are not obviously apparent based on the JIRA associated with the changes.
 Please avoid adding duplicate information across this changelog and JIRA/doc input pages.
 
+## [4.7.7]
+
+### Added Features
+
+### Removed Features
+
+### Deprecated Features
+
+### Technical Changes
+
+- ROX-30447, ROX-30498: Updated ROX_HASH_FLUSH_INTERVAL to allow for 0 such that hashes can be turned off entirely. If hashes are turned off, the table will be truncated thus freeing the space used by the table.
```

Comment:
allow for the value to be 0, such that ...

this freeing the used space.

Reply:
I suppose Tom took this from 4.8.3 notes, although without backticks formatting.
https://github.com/stackrox/stackrox/blob/release-4.8/CHANGELOG.md?plain=1#L22

Reply:
Misha assumed correctly. I have added the backticks and would prefer to keep the text same as for 4.8.3.